<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>CurrenciesWidget</class>
    <widget class="QWidget" name="CurrenciesWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QTableView" name="tableView"/>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QPushButton" name="addButton">
                            <property name="text">
                                <string>Add Currency</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="removeButton">
                            <property name="text">
                                <string>Remove Selected</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
